File: silver/2563/python/main.py

def setting(array, mX, mY):
    for y in range(10):
        for x in range(10):
            array[mY + y][mX + x] = True

if (__name__ == "__main__"):
    num = int(input())
    array = list()
    for i in range(100):
        array.append([False] * 100)
    for i in range(num):
        x, y = map(int, input().split())
        setting(array, x, y)
    
    count = 0
    for y in range(100):
        for x in range(100):
            if (array[y][x]):
                count += 1
    print(count)

File: silver/2563/c++/main.cpp
#include <iostream>

void create(bool array[][100], int x, int y)
{
    for (int i = y; i < y +10; i++) {
        for (int j = x; j < x +10; j++) {
            array[i][j] = true;
        }
    }    
}


int main()
{
    int num, x, y;
    bool array[100][100];
    
    for (int y = 0; y < 100; y++) {
        for (int x = 0; x < 100; x++) {
            array[y][x] = false;
        }
    }
    std::cin >> num;
    for (int i = 0; i < num; i++) {
        std::cin >> x >> y;
        create(array, x, y);
    }
    int count = 0;
    for (int y = 0; y < 100; y++) {
        for (int x = 0; x < 100; x++) {
            if (array[y][x]) {
                count++;
            }
        } 
    }
    std::cout << count;
    return 0;
}